config/config.exs

```diff
@@ -20,6 +20,15 @@ config :subwayside_ui, SubwaysideUiWeb.Endpoint,
   pubsub_server: SubwaysideUi.PubSub,
   live_view: [signing_salt: "Gjj0V7iV"]
 
+config :subwayside_ui, SubwaysideUi.GTFS.TrainCrowdingStatus,
+  feed_func: {SubwaysideUi.GTFS.TrainCrowdingStatus, :get_gtfs_realtime_feed, []}
+
+config :subwayside_ui, SubwaysideUi.GTFS,
+  # Which GTFS-RT (JSON) feed to load:
+  url: "https://mbta-gtfs-s3.s3.amazonaws.com/rtr/VehiclePositions_enhanced.json",
```

Comment:
> suggestion: this should probably be configured through the environment instead of hardcoded though I'm willing to hear either way. for one thing, this will mean that subwayside-ui-staging will show RTR prod data.

I agree - we only have one environment as-is, though. I added it as an optional override for the future 🙂